building/toolchains.rst

```diff
@@ -86,6 +99,5 @@ the following.
     $ ninja
     $ ninja install
 
-Now you'll have a llvm/clang toolchain ready to be used.
-
 .. _Arm GCC download page: https://developer.arm.com/open-source/gnu-toolchain/gnu-a/downloads
+.. _get_clang.sh: https://github.com/jforissier/docker_optee_os_ci_clangbuilt/blob/master/get_clang.sh
```

Comment:
Linking to this script is fine but I'd rather pin it to a known commit in case I later decide to remove the script or rename it or whatever. So the URL would be:

https://github.com/jforissier/docker_optee_os_ci_clangbuilt/blob/215d3ed2df20072dfc9140b361d3fee57eddcd19/get_clang.sh